public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -182,82 +175,84 @@ figure.image-display
 :marked
   ## Creating a class for the data
 
-  We are defining our data directly inside our component.
-  That's fine for a demo but certainly isn't a best practice. It's not even a good practice.
-  Although we won't do anything about that in this chapter, we'll make a mental note to fix this down the road.
+  You are defining your data directly inside the component which isn't best practice.
```

Comment:
which -> , which

(as a rule, use either "that" or ", which", depending on whether the phrase is restrictive—can't be dropped without changing the meaning—or nonrestrictive)